recipe/meta.yaml

```
@@ -1,13 +1,13 @@
-{% set version = "4.9.8" %}
-{% set build = 1 %}
+{% set version = "4.9.9-beta01" %}
```

```suggestion
{% set version = "4.9.9beta01" %}
```
And then hard-code the URL below rather than using the `{{ version }}` jinja2 variable.